discovery-provider/integration_tests/queries/test_notifications/test_save_of_repost_notification.py

```diff
@@ -85,17 +85,17 @@ def test_get_save_of_repost_notifications(app):
             assert len(user4_notifications) == 3
             assert_notification(
                 notification=user4_notifications[0],
-                group_id="save_of_repost:1:type:track",
+                group_id="save_of_repost:1:type:playlist",
```

Comment:
is this test flaking?? ill add hardcoded timestamps to make sure it returns notifs in the same order

Reply:
ooohhh wait is the change in order bc the change to the query you added ?